src/landing/LandingPage.js

```diff
@@ -24,6 +29,8 @@ const LandingPage = () => {
     if (queryData && usernameState && passwordState) {
       if (queryData.users.find((user) => user.username === usernameState
           && user.password === passwordState)) {
+        setUsername(usernameState);
+        setUserRole(3);
```

Comment:
This seems wrong, user role needs to be the one the comes from the API. What if an admin or a company user logs in ?

Reply:
fixed using the user.userRole there